contrib/vagrant/start.sh

```diff
@@ -84,17 +84,21 @@ function write_footer() {
     ipv6_addr="${2}"
     filename="${3}"
 
-    if [ -n "${IPV4}" ]; then
+    if [ -n "${IPV4}" ] && [ "${IPV4}" -ne "0" ]; then
         ipv4_options="--ipv4 --ipv4-range 10.${index}.0.1 "
     fi
 
+    if [ -n "${K8S}" ]; then
```

Comment:
No. For example I'm currently starting the script as:
```
IPV4=0 RELOAD=1 K8S=1 NFS=1 ./start.sh
```
where I don't want IPv4 but I want K8S